src/Symfony/Bundle/WebServerBundle/composer.json

```diff
@@ -17,9 +17,9 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/console": "~2.8.8|~3.0.8|~3.1.2|~3.2",
+        "symfony/console": "~3.3",
```

Comment:
there is no need to keep pre-3.3 deps for 3.3-introduced bundles IMHO, that makes things easier